### PR TITLE
Feature add serial reading

### PR DESCRIPTION
**Pull request by AdamBusch**
Add serial read functionality to server script. This will allow for us to get data from the CANdapter and parse messages from the car.
Also updated client view to better visualize what data we are getting from the car.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Wireless Telemetry Server
+Wireless telemetry server for Purdue Electric racing.
+Runs on a Raspberry Pi with a connected CANDapter and wireless router.
+[Wiki page](http://purdueelectricracing.com/wiki/index.php/Wireless_Telemetry)
```
Comment (bsmrdel) [inline README.md]: Wiki page looks great, thanks for good documentation on this.

**Review by bsmrdel — APPROVED** [review]

**Comment by RyanSchw** [comment]
Is this blocked by #5 ?

**Comment by AdamBusch** [comment]
> Is this blocked by #5 ?

No, I'm pretty sure that it passed all Travis checks, what do you mean?

**Comment by RyanSchw** [comment]
> > Is this blocked by #5 ?
> 
> No, I'm pretty sure that it passed all Travis checks, what do you mean?

Sorry, I'll clarify. You notice how #5 makes changes to `.travis.yml` for example? Well that change is also apparent in the diff for #6, since it was branched off #5. We didn't make those changes in this PR (pull request), we did it in the one you branched off of (#5). So that means #6 is blocked by #5 because you don't want the changes made in #5 showing up in the commit for #6.

We do something called **Squash and Merge** when accepting most pull requests which basically means take all of your commits within the PR and combine them into only one visible change within the `git log` on master. If we did that for #6 before #5, both changes would be squashed together and then we wouldn't be able to separate them.

**Comment by RyanSchw** [comment]
Can we pull master into this since #5 was just merged? After that's done, we can continue with the review.
@AdamBusch

**Review by RyanSchw — COMMENTED** [review]
Why are we including these JS files (specifically, `randomColor.js` and `smoothie.js`) to the repo? Is telemetry supposed to be able to function offline? Why does the user have to download these files instead of pulling from a CDN?

**Comment by AdamBusch** [comment]
> Why are we including these JS files (specifically, `randomColor.js` and `smoothie.js`) to the repo? Is telemetry supposed to be able to function offline? Why does the user have to download these files instead of pulling from a CDN?

This is fixed in #7, that was an old experimental dashboard that I was prototyping. But yes telemetry should be able to run offline so I did not use a CDN.

File: serial-read.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import serial
+import time
+import datetime
+
+ser = serial.Serial('/dev/ttyUSB0', 2500000, timeout=None)
```
Comment (RyanSchw) [inline serial-read.py]: Is this going to be the same every time?

**Review by RyanSchw — APPROVED** [review]
Looks good. At some point, you may want to consider moving your JS files to a `/static` directory for better organization.

File: README.md (hunk 1 above)
Comment (RyanSchw) [reply README.md]: Seconded!

**Pull request merged**